templates/module/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "{{project-name}}"
-version = "0.3.3"
+version = "0.0.1"
```

Comment:
It didn't make much sense to me that we were generating projects and modules that had the same version as the micro-rdk from which they were generated. If you disagree I can drop this out easily enough.

I'd filed it as https://viam.atlassian.net/browse/RSDK-9696 but we hadn't got around to triaging it yet. Still, this seems as good a time as any, and it will mean one less version we need to hand-edit on each release (until we have a more automated release process).

Reply:
I agree.